core/src/main/java/org/apache/spark/memory/SparkOutOfMemoryError.java

```diff
@@ -51,4 +51,6 @@ public String getErrorClass() {
     public String getSqlState() {
         return SparkThrowableHelper.getSqlState(errorClass);
     }
+
+    public boolean isInternalError() { return SparkThrowableHelper.isInternalError(errorClass); }
```

Comment:
do we still need it after we have the default implementation?

Reply:
Good catch, I forgot these.